CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ guide](./docs/migration-guide.md) for details.
 
 - [ENHANCEMENT] Add the option to log to stdout instead of a Loki instance. (@joe-elliott)
 
+- [ENHANCEMENT] Update Prometheus dependency to v2.27.0 and add exemplars support (@mapno)
```

Comment:
Also, can you update the Prometheus entry in .github/depcheck.yml?

Reply:
Updated in 92c0e4c8cd9e2e517323f0e0a1e38a922feea77c